README.md

```diff
@@ -1,3 +1,64 @@
-# Contributing
+# Microsoft Azure Key Vault SDK for Java
 
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+This is the Microsoft Azure Key Vault client library which allows for the consumption of Key Vault services.
+
+## Sample code
+You can find sample code that illustrates key vault usage scenarios [here](https://azure.microsoft.com/en-us/resources/samples/?sort=0&service=key-vault&platform=java).
+
+## Download
+
+To get the binaries of this library as distributed by Microsoft, ready for use within your project, you can use Maven.
+
+```xml
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-keyvault</artifactId>
+    <version>1.0.0</version>
+</dependency>
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-keyvault-cryptography</artifactId>
+    <version>1.0.0</version>
+</dependency>
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-keyvault-extensions</artifactId>
+    <version>1.0.0</version>
+</dependency>
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-keyvault-core</artifactId>
+    <version>1.0.0</version>
+</dependency>
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-keyvault-webkey</artifactId>
+    <version>1.0.0</version>
+</dependency>
+```
+
+## Pre-requisites
+- A Java Developer Kit (JDK), v 1.7 or later
+- Maven
+- Azure Service Principal, using
+[Azure CLI](http://azure.microsoft.com/documentation/articles/resource-group-authenticate-service-principal-cli/),
+[PowerShell](http://azure.microsoft.com/documentation/articles/resource-group-authenticate-service-principal/)
+or [Azure Portal](http://azure.microsoft.com/documentation/articles/resource-group-create-service-principal-portal/).

```

Comment:
I don't think we need to include this here.  You don't need a SP to interact with the repo.